arch/x86/soc/apollo_lake/linker.ld

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2011-2014, Wind River Systems, Inc.
```

Comment:
Intel 2018?

Reply:
The file was copied from the arch/x86/soc/atom. Nothing changed so preserve the original copyright.